### DISCUSSION TITLE
Welcome to browser Discussions!

### DISCUSSION THREAD

**Discussion by plateaukao** [plateaukao/einkbro#28] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dredmorbius** [comment]
I've just filed a number of issues, though the reason I've done so is _because_ EInkBro is so darned useful.

Thanks for a great tool, really appreciate the design choices, especially the paginated reading and controls.  For website reading on my e-reader, this is far and away my preferred option for long-form material.
